Centaurus.Models/Effects/EffectsContainer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Centaurus.Xdr;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.Models
+{
+    [XdrContract]
+    public class EffectsContainer
```

Comment:
Do you really need this contract? `List<Effect>` can be used implicitly where it's needed. Please try to avoid unneeded contracts as every contract serializer has performance penalty.

Reply:
I need it to compute effects hash, but our xdr serializer supports only classes with XdrContractAttributes